<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCommentDlg</class>
 <widget class="QDialog" name="AddCommentDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add A Comment</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="Cancel_Add_Comments">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_Add_Comment">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>95</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string> Add Comments</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>9</y>
     <width>411</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Select Forum ID,Topic ID and Thread ID</string>
      </property>
      <widget class="QWidget" name="layoutWidget4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>361</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Select Topic ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Select_Topic_ID">
          <property name="placeholderText">
           <string>Select Topic ID</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>361</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Select_Thread_ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Select_Thread_ID">
          <property name="placeholderText">
           <string>Select Thread ID</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>361</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Select Forum ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Select_Forum_ID">
          <property name="placeholderText">
           <string>Select Forum ID</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Add Comments</string>
      </property>
      <widget class="QTextEdit" name="Add_Comments_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>371</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
